src/Gui/NaviCube.cpp

```diff
@@ -454,9 +452,10 @@ void NaviCubeImplementation::createCubeFaceTextures() {
             delete m_LabelTextures[pickId].texture;
         }
         m_LabelTextures[pickId].texture = new QOpenGLTexture(image.mirrored());
-        m_LabelTextures[pickId].texture->generateMipMaps();
```

Comment:
do we now if this is safe to use for example with noveau, old cards, arm machines?

Reply:
I think you ment the line with: `m_LabelTextures[pickId].texture->setMaximumAnisotropy(4.0);`?
There should be a check built in QT but there seems to be a bug reported twice but not resolved. https://bugreports.qt.io/browse/QTBUG-47091 and https://bugreports.qt.io/browse/QTBUG-44604
I could do this to be safer: 
```
if (m_LabelTextures[pickId].texture->hasFeature(QOpenGLTexture::AnisotropicFiltering))
    m_LabelTextures[pickId].texture->setMaximumAnisotropy(4.0);
```

Reply:
The extension is listed as ubiquitous here: https://www.khronos.org/opengl/wiki/Ubiquitous_Extension

Reply:
excellent, should be acceptable then